cms/templates/settings.html

```diff
@@ -199,6 +199,26 @@ <h3 class="title">${_("These Dates Are Not Used When Promoting Your Course")}</h
           </div>
           % endif
         </section>
+
+        <section class="group-settings details">
```

Comment:
Just to be clear, we want this "Course Language" field to show up regardless of whether or not the course is running on our edX Drupal site? There are a lot of section on this page that are only shown if "about_page_editable" is True (which means NOT running on edX Drupal site), and I want to make sure this language dropdown should not be in that category.